### PR TITLE
Results vis

### PR DESCRIPTION
**Pull request by daler**
Replacing #26 with this PR. The advantage of having a branch off this repo (rather than forks) is so that anyone with edit access to this repo can contribute/fix the PR directly, rather than asking the original submitter to make the changes.

### CODE REVIEW

**Comment by daler** [comment]
(also see #24)

**Comment by khughitt** [comment]
Good call. I've mostly worked with forking model for collaboration in the past, but I can see the advantage of working with branches of main repo.

**Comment by khughitt** [comment]
Ah, thanks for catching that!

You can also add `options(knitr.duplicate.label='allow')` to the top of the Rmd to allow duplicate labels by default. This is something I usually tend to enable.

**Comment by khughitt** [comment]
@daler Updated include changes described in #29.

**Comment by khughitt** [comment]
Added dependency: [heatmaply ](https://github.com/talgalili/heatmaply).

**Comment by daler** [comment]
Getting an error, 

```
Quitting from lines 224-238 (visualize_results.Rmd) 
Error in eval(expr, envir, enclos) : 
  could not find function "rownames_to_column"
```

(doesn't seem to be defined anywhere)

**Comment by daler** [comment]
Ah wait, looks like it's in `tibble` which is added in your last commit. Retrying . . .

**Comment by khughitt** [comment]
Yea, sorry about that -- tibble should be installed if you have an up-to-date `dplyr` on the system. Otherwise you can `install.packages()` it.

File: reports/templates/R/results_helper_functions.R [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+    # iterate over P3 outputs
+    for (i in 1:length(infiles)) {
+        # load result
+        infile <- infiles[i]
+        message(sprintf("Loading %s.", infile))
+        load(infile)
+
+        # drug name
+        drugs <- c(drugs, p3_result$drug)
```
Comment (daler) [inline reports/templates/R/results_helper_functions.R]: @khughitt should be `c(drugs, p3_result$drug_name)` according to `tools/post_processing.R`?

**Comment by daler** [comment]
@khughitt it's been a while . . . I'm going to merge this and then work on any issues that arise after the fact.

**Pull request merged**